python/lsst/dax/metaserv/metaAdminImpl.py

```diff
@@ -247,12 +249,12 @@ def addInstitution(self, name):
 
         @param name  the name
         """
-        db = Db(read_default_file=self._msMysqlAuthF)
-        ret = db.execCommand1(
+        conn = getEngineFromFile(self._msMysqlAuthF)
```

Comment:
I would prefer we call this `engine` and not connection so it's clear what we're dealing with.